projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli.py

```diff
@@ -178,7 +178,13 @@ def deploy(
         # default to ask for job version to update
         if not vdk_version and not job_version and enabled is None:
             job_version = get_or_prompt("Job Version", job_version)
-        return cmd.update(name, team, enabled, job_version, vdk_version, output)
+        if job_path:
+            reason = get_or_prompt("Reason", reason)
+            return cmd.create(
+                name, team, job_path, reason, output, vdk_version, enabled
```

Comment:
Until now you can do `vdk deploy -p xxx` and you'd be promoted for reason (handled by `if operation == CREATE` below)

But you could not do `vdk deploy -p xxx --disable` . You could either do `vdk deploy --disable` or `vdk deploy -p xxx` 

And now I want both to be possible. I am trying to make the command as smart as possible so whatever the user passes it would work .